src/components/WorkPoolTypeSelect.vue

```diff
@@ -13,9 +13,10 @@
   import { useWorkspaceApi } from '@/compositions'
   import { titleCase } from '@/utilities'
 
-  const props = defineProps<{
+  const props = withDefaults(defineProps<{
     selected: string | null | undefined,
-  }>()
+    disabled: boolean,
+  }>(), { disabled: false })
```

Comment:
If you use a `boolean` prop, you don't need to [specify a default](https://vuejs.org/guide/components/props.html#boolean-casting)

Reply:
I got a type error if I didn't specify a default, so I make `disabled` optional in the props typing. Is that standard when declaring `boolean` props?

Reply:
An optional boolean defaults to `false` automatically. You probably got a type error when you went to use this component? Making it optional is preferred over setting a default value. 

More context from the [docs](https://vuejs.org/guide/components/props.html#prop-validation)
> The Boolean absent props will be cast to false. You can change this by setting a default for it — i.e.: default: undefined to behave as a non-Boolean prop.